leapp/actors/__init__.py

```diff
@@ -206,7 +191,8 @@ def get_folder_path(self, name):
         :return: Found folder path
         :rtype: str or None
         """
-        return self._get_folder_path(self.files_paths, name)
+        from leapp.libraries.stdlib import path
```

Comment:
If you cannot do it globally, because of circular imports, I'd probably import it once in __init__ and assign it to self._path, WDYT?

Reply:
to be honest I don't really like any option in "Inplace import and code duplication" vs "adding a rather obscure instance attribute". 
But probably your way is better.